app/services/plugin_service.py

```
@@ -0,0 +1,61 @@
+        res = supabase.table(PLUGIN_TABLE).update(data).eq("id", config_id).eq("user_id", user_id).execute()
+        if res.data:
+            return res.data[0]
+        return None
+
+    @staticmethod
+    async def delete_plugin_config(config_id: str, user_id: str) -> bool:
+        res = supabase.table(PLUGIN_TABLE).delete().eq("id", config_id).eq("user_id", user_id).execute()
+        return bool(res.data)
+
+    @staticmethod
+    async def list_plugin_configs(user_id: str) -> List[dict]:
```

Ruff (0.8.2)

Use `list` instead of `List` for type annotation

Replace with `list`

(UP006)